packages/trader/src/sass/app/modules/trading-mobile.scss

```diff
@@ -115,6 +115,41 @@
             }
         }
     }
+    .barrier__modal {
+        &-container {
+            display: flex;
+            justify-content: space-between;
+            .strike--value-container {
+                margin-top: 0.5rem;
+                margin-right: 2.4rem;
+            }
+            .trade-container__price-info-currency {
```

Comment:
I can't remove text style (line 127-129) by using `<Text />` component, as `.trade-container__price-info-currency` is already used in `<ValueMovement />` (it is being passed as className in <Money />)